crates/algod_client/src/apis/abort_catchup.rs

```diff
@@ -0,0 +1,96 @@
+/*
+ * Algod REST API.
+ *
+ * API endpoint for algod operations.
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Contact: contact@algorand.com
+ * Generated by: Rust OpenAPI Generator
+ */
+
+use algokit_http_client::HttpClient;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+use super::{AlgodApiError, ContentType, Error};
+
+// Import all custom types used by this endpoint
+use crate::models::{AbortCatchup, ErrorResponse};
+
+// Import request body type if needed
+
+/// struct for typed errors of method [`abort_catchup`]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum AbortCatchupError {
+    Status400(ErrorResponse),
+    Status401(ErrorResponse),
+    Status500(ErrorResponse),
+    Statusdefault(),
+    DefaultResponse(),
+    UnknownValue(serde_json::Value),
+}
+
+/// Given a catchpoint, it aborts catching up to this catchpoint
+pub async fn abort_catchup(
+    http_client: &dyn HttpClient,
+    catchpoint: &str,
+) -> Result<AbortCatchup, Error> {
+    let p_catchpoint = catchpoint;
+
+    let use_msgpack = false;
+
+    let path = format!(
+        "/v2/catchup/{catchpoint}",
+        catchpoint = crate::apis::urlencode(p_catchpoint)
+    );
+
+    let query_params: HashMap<String, String> = HashMap::new();
+
+    let mut headers: HashMap<String, String> = HashMap::new();
+    if use_msgpack {
```

Comment:
The `use_msgpack` logic (and `use_msgpack` itself) could be remove and become logic in the jinja template itself.